composer.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "reload/bugyield",
+  "description": "A script to sync between Harvest and Jira.",
+  "license": "MIT",
+  "autoload" : {
+    "psr-4": {
+      "": "src/"
```

Comment:
Do you mean move the directory to the root and completely skip the "src" dir? In that case, won't the "bugyield" unix executable file cause errors with a directory with the same name? I believe that was my experience (but my memory might be mistaken though).